automated/linux/kselftest/kselftest.sh

```diff
@@ -200,6 +200,12 @@ elif [ -n "${TST_CMDFILES}" ]; then
         echo "============== Tests to run ==============="
         cat shardfile
         echo "===========End Tests to run ==============="
+        if [ -s shardfile ]; then
+            report_pass "shardfile-${TST_CMDFILES}"
```

Comment:
this should be ${test} and not ${TST_CMDFILES} or?

Reply:
${TST_CMDFILES} is arm64
${test} is arm64:fp-stress

We are trying to detect the when sharding file size is zero that mean the total subtests get missed.

Reply:
@roxell  Thanks for the review comments.
Now those review comments addressed and pushed new version of patch.